test/goHome.test.ts

```diff
@@ -0,0 +1,55 @@
+import { chromium, Page, Browser, BrowserContext } from "playwright"
+
+// NOTE: this is hard-coded and passed as an environment variable
+// See the test job in ci.yml
+const PASSWORD = "e45432jklfdsab"
+
+describe("login", () => {
+  let browser: Browser
+  let page: Page
+  let context: BrowserContext
+
+  beforeAll(async () => {
+    browser = await chromium.launch()
+    context = await browser.newContext()
+  })
+
+  afterAll(async () => {
+    await browser.close()
+    await context.close()
+  })
+
+  beforeEach(async () => {
+    page = await context.newPage()
+  })
+
+  afterEach(async () => {
+    await page.close()
+    // Remove password from local storage
+    await context.clearCookies()
+  })
+
+  it("should see a 'Go Home' button in the Application Menu that goes to coder.com", async () => {
+    await page.goto("http://localhost:8080")
+    // In case the page takes a long time to load
+    await page.waitForTimeout(2000)
```

Comment:
Playwright doesn't wait for the page to finish loading?  Maybe we need something like https://playwright.dev/docs/navigations/#custom-wait

Reply:
That worked like a charm! Good find